backend/src/StamAcasa.Api/Controllers/AssessmentController.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using IdentityModel;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using StamAcasa.Api.Services;
+using StamAcasa.Common.DTO;
+using StamAcasa.Common.Services;
+
+
+namespace StamAcasa.Api.Controllers
+{
+    [Authorize(AuthenticationSchemes = "answersApi")]
+    [Route("api/[controller]/{version}")]
+    [ApiController]
+    public class AssessmentController : ControllerBase
```

Comment:
Can we merge this with the `FormController`?
The `FormController` already handles returning all the assessments that a user has filled in. 
The initial idea was to add a new endpoint `/api/form/version` that would return the version of the form the user needs to have filled.

Reply:
I've deleted this pr and made a new one with your request. In the new one, there is no more database modification because we don't really need to store the two son files there.